### ISSUE TITLE
Rule Proposal: vue/no-constant-condition

### ISSUE THREAD

**Issue by Zzzen** [vuejs/eslint-plugin-vue#1358]
**Please describe what the rule should do:**
like [no-constant-condition](https://eslint.org/docs/rules/no-constant-condition) in eslint, but works with `v-if`/`v-show`.

**What category should the rule belong to?**

[ ] Enforces code style (layout)
[x] Warns about a potential error (problem)
[ ] Suggests an alternate way of doing something (suggestion)
[ ] Other (please specify:)

**Provide 2-3 code examples that this rule should warn about:**

```vue
<div v-if="true"></div>
<div v-if="true || prop1"></div>
```

**Additional context**
